src/reason-parser/reason_parser.mly

```diff
@@ -1061,6 +1061,47 @@ let filter_raise_spread_syntax msg nodes =
         raise Reason_syntax_util.(Error(dotdotdotLoc, (Syntax_error msg)))
     | None -> node
     ) nodes
+
+let err loc s =
+  raise Reason_syntax_util.(
```

Comment:
Do we not have a similar utility already? Genuine question. We should have a single one that throws a single type of exception